sdk/core/Azure.Core.Amqp/src/Azure.Core.Amqp.csproj

```diff
@@ -7,15 +7,19 @@
     <ApiCompatVersion>1.3.0</ApiCompatVersion>
     <PackageTags>Microsoft Azure AMQP</PackageTags>
     <Nullable>enable</Nullable>
-    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <TargetFrameworks>$(RequiredTargetFrameworks);net6.0</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="System.Memory" />
     <PackageReference Include="System.Memory.Data" />
     <PackageReference Include="Microsoft.Azure.Amqp" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(TargetFramework)' != 'net6.0'">
+    <!-- The APIs of these packages are inbox on modern .NET -->
+    <PackageReference Include="System.Memory" />
+  </ItemGroup>
+
```

Comment:
Or if we want to avoid the multitargeting, we can entirely remove the direct dependency and let `Azure.Core` provide it. What do you think?